styles.css

```diff
@@ -914,4 +914,204 @@ ul {
   .hamburger.active .line:nth-child(3) {
     transform: translateY(-8px) rotate(-45deg);
   }
+
+  .desktop-only {
+    display: none;
+  }
+
+  .pop-up {
+    width: 70%;
+    top: 20%;
+    left: 30%;
+    height: 80vh;
+    transform: translate(-50%, -50%);
+    z-index: 100;
+  }
+
+  .pop-up-btn-box.desktop-only {
+    display: none;
+  }
+
+  .pop-up-close-btn {
+    margin-right: -16px;
+    height: 2rem;
+    border-radius: 3px;
+    padding: 0;
+    position: absolute;
+    top: 1.3%;
+    right: 4.6%;
+  }
+
+  .mobile-only .btn span {
+    font-size: 14px;
+  }
+
+  .pop-up-heading {
+    font-size: 2rem;
+    font-weight: 600;
+  }
+
+  .pop-up-technologies button {
+    font-size: 1rem;
+    font-weight: 400;
+  }
+
+  .pop-up-description {
+    display: flex;
+    flex-direction: column;
+    gap: 0.6rem;
+  }
+
+  .pop-up-description p {
+    font-size: 1.2rem;
+    font-weight: 400;
+    line-height: 2rem;
+    text-align: left;
+  }
+}
+
+.pop-up {
+  scrollbar-gutter: stable;
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  gap: 0.6rem;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  height: 75vh;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  background-color: #fff;
+  border-radius: 1rem;
+}
+
+.pop-up-img-box {
+  border-radius: 1.6rem;
+  position: relative;
+}
+
+.pop-up-hero-img {
+  width: 100%;
+}
+
+.pop-up-close-btn {
+  cursor: pointer;
+  background-color: #ebecf0;
+  float: right;
+}
+
+.pop-up-technologies {
+  display: flex;
+  gap: 5px;
+}
+
+.pop-up-technologies button {
+  border-radius: 4px;
+  padding: 8px 12px 8px 12px;
+  letter-spacing: 0.03em;
+  text-align: left;
+}
+
+.pop-up-btn-box {
+  display: flex;
+  gap: 1.6rem;
+  justify-content: flex-end;
+  justify-items: center;
+}
+
+.btn {
+  background-color: #36b37e;
+  color: white;
+  height: 40px;
+  border: none;
+  border-radius: 6px;
+}
```

Comment:
- [ ] Kindly, make sure to match the Figma design as much as possible. Currently, your popup buttons are missing enough padding.

<table>
<tr>
<th>Your design</th>
<th>Figma</th>
</tr>
  <tr>
    <td><img src="https://github.com/tajulafreen/Portfolio/assets/86069740/4b22b792-f35e-4c52-9efd-f292ef232205"/></td>
    <td><img src="https://github.com/tajulafreen/Portfolio/assets/86069740/c90c386f-9966-4319-b6cf-8a5544c0229e"/></td>
  </tr>
</table>